tools/build-tools/src/repoPolicyCheck/handlers/assertShortCode.ts

```diff
@@ -0,0 +1,112 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    Node,
+    NumericLiteral,
+    Project,
+    SourceFile,
+    StringLiteral,
+    SyntaxKind,
+    TemplateLiteral,
+} from "ts-morph";
+import {Handler} from "../common";
+
+const shortCodes = new Map<number, Node>();
+const newAssetFiles = new Set<SourceFile>();
+let maxShortCode = -1;
+
+function getCallsiteString(msg: Node){
+    return `${msg.getSourceFile().getFilePath()}@${msg.getStartLineNumber()}`
+}
+
+function getAssertMessageParams(sourceFile: SourceFile): (StringLiteral | NumericLiteral | TemplateLiteral)[]{
```

Comment:
Would be great to sprinkle code with comments here and there

Reply:
can do